tests/unit/dataactvalidator/test_a1_appropriations.py

```diff
@@ -56,4 +56,4 @@ def test_failure(database):
 
     # Non-overlapping ranges of agency IDs should generate two errors
     errors = number_of_errors(_FILE, database, models=[tas, tas_null, approp, approp_null])
-    assert errors == 2
\ No newline at end of file
+    assert errors == 2
```

Comment:
Flake8 has a rule that all files must end in a new line, but we're actually ignoring that at the moment. The rule stems from the history of Python: part of the posix standard is for all lines in a text file (including the last) to end in a newline character. This also means that unixy text editors (including vim) add the character automatically. I mass edited the unit tests (as several had unnecessary imports), so vim added the extra newline; I figured that it'd be easier to commit the changes (and prep for the flake8 rule) than undo them.